introduction.md

```diff
@@ -2,4 +2,5 @@ Hello. This is UlyssesOdysseus.
  My favorite path in code school is HTML.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?